ibis/backends/duckdb/__init__.py

```diff
@@ -1062,7 +1062,9 @@ def read_mysql(
 
         return self.table(table_name, database=(database, catalog))
 
-    def read_sqlite(self, path: str | Path, table_name: str | None = None) -> ir.Table:
+    def read_sqlite(
+        self, *, path: str | Path, table_name: str | None = None
```

Comment:
```suggestion
        self, path: str | Path, *, table_name: str | None = None
```

Reply:
same comment than above, we made `uri` required in `read_mysql`